Task3/js/app.js

```diff
@@ -0,0 +1,109 @@
+const container = document.querySelector('.zombies');
+const status = document.querySelector('.zombies-status');
+const numberZombie = document.querySelectorAll('.zombies-status span')[0];
+const allZombies = document.querySelectorAll('.zombies-status span')[1];
+let counter = 0;
+let zombie;
+
+allZombies.textContent = zombies.length;
+numberZombie.textContent = counter + 1;
+
+function Zombie() {
+    this.health = 0;
+    this.healthCurrent = 0;
+    this.healthPercent = 100;
+    this.hitDamage = HIT_DAMAGE;
+
+    this.getDamage = () => {
+        if (this.healthCurrent > 0) {
+            this.healthCurrent -= this.hitDamage;
+        }
+
+
+        this.calculateHealth();
+    };
+
+    this.calculateHealth = () => {
+        this.healthPercent = this.healthCurrent / this.health  * 100;
+
+        if (this.healthPercent < 0) {
+            this.healthPercent = 0;
+        }
+    }
+}
+
+function SmallZombie() {
```

Comment:
please move the all class-functions SmallZombie, MadZombie, StrongZombie in the separate file (one file for each zombie type)